README.md

```diff
@@ -1,7 +1,56 @@
-# Scoop [![Build status](https://ci.appveyor.com/api/projects/status/05foxatmrqo0l788?svg=true)](https://ci.appveyor.com/project/lukesampson/scoop) [![Gitter chat](https://badges.gitter.im/lukesampson/scoop.png)](https://gitter.im/lukesampson/scoop)
+<p align="center">
+<img src="scoop.png" alt="Long live Scoop!"/>
```

Comment:
Is this the logo we’ve settled on?

Reply:
ummm, no.
@lukesampson should settle this one.